README.md

```diff
@@ -118,8 +120,177 @@ Following feautres/pages exist in the codebase but have been disabled for now (c
 - Search (both global and blog-based)
 - Awards
 
-## License
+## Blog Post Contribution Guide
+
+Welcome to the **Blog Post Contribution Guide**! This guide provides step-by-step instructions for adding a new blog post to this project.
+
+---
+
+## Table of Contents
+
+1. [Project Setup](#setup-and-installation)
+2. [Steps for Adding a New Blog Post](#steps-for-adding-a-new-blog-post)
+   - [1. Add an `id` for the Blog Post](#1-add-an-id-for-the-blog-post)
+   - [2. Add a High-Quality Blog Image](#2-add-a-high-quality-blog-image)
+   - [3. Add a Small Blog Image](#3-add-a-small-blog-image)
+   - [4. Add the Post Date](#4-add-the-post-date)
+   - [5. Assign a Category](#5-assign-a-category)
+   - [6. Add a Title](#6-add-a-title)
+   - [7. Add Tags](#7-add-tags)
+   - [8. Add an Icon](#8-add-an-icon)
+   - [9. Add the Author's Name](#9-add-the-authors-name)
+   - [10. Add the Author's Quote](#10-add-the-authors-quote)
+   - [11. Add the Author's Image](#11-add-the-authors-image)
+   - [12. Add the Author's Profile](#12-add-the-authors-profile)
+   - [13. Add Blog Content](#13-add-blog-content)
+   - [14. Add Blog Sections](#14-add-blog-sections)
+3. [Final Example BlogPost Object](#final-example-blogpost-object)
+
+---
+
+## Steps for Adding a New Blog Post
+
+### 1. Add an id for the Blog Post
+Assign an id that is one value greater than the previous blog post. If the previous Blog id is `1` use `2` Example:
+```javascript 
+id: 2,
+```
+
+### 2. Add a High-Quality Blog Image
+Upload an image (dimensions: **1080x600**) to the directory `public/assets/img/blog`. Add the image path:
+```javscript 
+image: '/assets/img/blog/newBlog2.jpg', 
+```
+
+### 3. Add a Small Blog Image
+Upload a smaller version (dimensions: **180x140**) and add its path:
+```javascript 
+smallImageUrl: '/assets/img/blog/newMiniBlog2.jpg', 
+```
+
+### 4. Add the Post Date
+Use the format `Month DD, YYYY`. Example:
+```javascript 
+date: 'November 1, 2024', 
+```
+
+### 5. Assign a Category
+Choose an existing category from `data/categories-tags.js`. If you want to add one of your own add it to the `categories` list and then use it. Example:
+```javascript
+ category: 'category', 
+```
+
+### 6. Add a Title
+Add a descriptive title. Avoid symbols or random characters. Example:
+```javascript
+ title: 'Your New Blog Post', 
+```
+
+### 7. Add Tags
+Include at least one tag as an array, choose tags form the tags list in `data/categories-tags.js` or add your own tags to the list and then use it. Example:
+```javascript
+tags: ['Tag1', 'Tag2', 'Tag3'],
+ ```
+
+### 8. Add an Icon
+Use the standard icon path, you don't need a new one:
+```javascript
+icon: '/assets/img/icon/arrow-left-top.svg',
+ ```
+
+### 9. Add the Author's Name
+Example:
+```javascript
+author: 'Author 1', 
+```
+
+### 10. Add the Author's Quote
+Example:
```

Comment:
Explain what is this, why is it required, and if it is an optional field (add this for rest of the values as well to explain if some field is optional)

Reply:
Okay